test/metabase/test/data/sql_jdbc/load_data.clj

```diff
@@ -194,16 +193,18 @@
         (log/debugf "Setting timezone to UTC before inserting data with SQL \"%s\"" set-timezone-sql)
         (jdbc/execute! spec [set-timezone-sql])))
     (mt/with-database-timezone-id nil
-      (try
-        ;; TODO - why don't we use [[execute/execute-sql!]] here like we do below?
-        (doseq [sql+args statements]
-          (log/tracef "[insert] %s" (pr-str sql+args))
-          (jdbc/execute! spec sql+args {:set-parameters (fn [stmt params]
-                                                          (sql-jdbc.execute/set-parameters! driver stmt params))}))
-        (catch SQLException e
-          (log/error (u/format-color 'red "INSERT FAILED: \n%s\n" (pr-str statements)))
-          (jdbc/print-sql-exception-chain e)
-          (throw e))))))
+      (doseq [sql-args statements]
+        (log/tracef "[insert] %s" (pr-str sql-args))
+        (try
```

Comment:
I moved the `try-catch` to be around the individual `execute!` calls instead of the whole thing, because it's much easier to debug the specific one that's failing